src/accessibility/KeyboardShortcuts.ts

```diff
@@ -713,4 +719,19 @@ export const KEYBOARD_SHORTCUTS: IKeyboardShortcuts = {
             key: Key.COMMA,
         },
     },
+    [KeyBindingAction.NextLandmark]: {
+        default: {
+            ctrlOrCmdKey: true,
+            key: Key.F6,
+        },
+        displayName: _td("keyboard|next_landmark"),
+    },
+    [KeyBindingAction.PreviousLandmark]: {
+        default: {
+            ctrlOrCmdKey: !IS_ELECTRON,
```

Comment:
What's the reason for this being dependent on Electron, but not `NextLandmark` above?

Reply:
That's an oversight, thanks! This is similar to Slack where in the app pressing cmd is not necessary.